acq_stat_reports/acq_stat_reports.py

```diff
@@ -38,18 +43,29 @@ def get_parser():
     parser.add_argument(
         "--datadir",
         default=Path(SKA / "data" / "acq_stat_reports"),
-        help="Output directory",
+        help="Output directory (default is $SKA/data/acq_stat_reports)",
         type=Path,
     )
     parser.add_argument(
         "--webdir",
         default=Path(SKA / "www" / "ASPECT" / "acq_stat_reports"),
-        help="Output directory",
+        help="Output directory (default is $SKA/www/ASPECT/acq_stat_reports)",
         type=Path,
     )
-    parser.add_argument("--url", default="/mta/ASPECT/acq_stat_reports/")
-    parser.add_argument("--start_time", default=None)
-    parser.add_argument("--days_back", default=30, type=int)
+    parser.add_argument(
+        "--url",
+        help="URL root of the deployed page (default is /mta/ASPECT/acq_stat_reports/)",
+        default="/mta/ASPECT/acq_stat_reports/",
+    )
+    parser.add_argument(
+        "--start_time", help="Start time (default is NOW)", default=None
+    )
+    parser.add_argument(
+        "--days_back",
+        help="How many days to look back (default is 30)",
+        default=30,
+        type=int,
+    )
     parser.add_argument(
         "-v",
```

Comment:
Out of scope for this PR, but modern code should use `--log-level`. At some point we should change this.

Reply:
I added this change to #9